src/crypto/muhash.cpp

```diff
@@ -0,0 +1,307 @@
+// Copyright (c) 2017 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <crypto/muhash.h>
+
+#include <crypto/chacha20.h>
+#include <crypto/common.h>
+
+#include <assert.h>
+#include <stdio.h>
+
+#include <limits>
+
+namespace {
+
+/** 2^3072 - 1103717, the next largest 3072-bit safe prime number, is used as the order of the group. */
```

Comment:
'next' from what? Is this just the largest 3072 bit safe prime number?

Reply:
Weird, I don't know how that made it in there, removed `next`